src/main/org/pantsbuild/jarjar/util/StandaloneJarProcessor.java

```diff
@@ -52,7 +52,7 @@ public static void run(File from, File to, JarProcessor proc) throws IOException
                     } else if (struct.name.endsWith("/")) {
                         // TODO(chrisn): log
                     } else {
-                        throw new IllegalArgumentException("Duplicate jar entries: " + struct.name);
+                        throw new DuplicateJarEntryException(from.getAbsolutePath(), struct.name);
```

Comment:
It seems like the main runner is broken anyhow: https://github.com/jsirois/jarjar/blob/master/src/main/org/pantsbuild/jarjar/MainUtil.java#L41

Your fix here works, but the main runner should be exiting non-zero on a syntax error (IAE) after logging.